couchdb/server/mime.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+#
+import logging
+
+from pprint import pformat
+
+from couchdb.server.exceptions import Error
+from couchdb.util import OrderedDict
+
+log = logging.getLogger(__name__)
+
+__all__ = ('best_match', 'MimeProvider', 'DEFAULT_TYPES')
+
+
+# Some default types.
+# Build list of `MIME types
+# <http://www.iana.org/assignments/media-types/>`_ for HTTP responses.
+# Ported from `Ruby on Rails
+# <https://github.com/rails/rails/blob/v3.1.0/actionpack/lib/action_dispatch/http/mime_types.rb>`_
+DEFAULT_TYPES = {
+    'all': ['*/*'],
+    'text': ['text/plain; charset=utf-8', 'txt'],
+    'html': ['text/html; charset=utf-8'],
+    'xhtml': ['application/xhtml+xml', 'xhtml'],
```

Comment:
Good idea. May be JIRA is on maintenance or something? Have you change your keyboard layout before typing a password?